docs/custom_loading.rst

```diff
@@ -161,16 +161,16 @@ This again will be done in a separate python file and placed in the user's
 
     @custom_writer("fits-writer")
     def generic_fits(spectrum, file_name, **kwargs):
-        flux = spectrum.flux.value
-        disp = spectrum.spectral_axis.value
+        flux = spectrum.flux
+        disp = spectrum.spectral_axis
         meta = spectrum.meta
 
         tab = Table([disp, flux], names=("spectral_axis", "flux"), meta=meta)
 
         tab.write(file_name, format="fits")
 
 Please note that it is assumed that the axis (of the ``flux`` and ``spectral_axis``)
```

Comment:
```suggestion
Note that this will store the units of the ``flux`` and ``spectral_axis`` as attributes of the ``Quantity`` columns in the ``Table``, which is saving them automatically to the FITS file.
```